src/integration/meson.ts

```diff
@@ -0,0 +1,25 @@
+import { Manifest } from '../manifest'
+import { Integration } from './base'
+
+export class Meson extends Integration {
+    isApplicable(manifest: Manifest): boolean {
+        return manifest.module().buildsystem === 'meson'
+    }
+
+    async load(manifest: Manifest): Promise<void> {
+        await manifest.overrideWorkspaceConfig('mesonbuild', 'configureOnOpen', false)
+
+        const buildSystemBuildDir = manifest.buildSystemBuildDir()
+        if (buildSystemBuildDir !== null) {
+            await manifest.overrideWorkspaceConfig('mesonbuild', 'buildFolder', buildSystemBuildDir)
+        }
+
+        await manifest.overrideWorkspaceCommandConfig('mesonbuild', 'mesonPath', 'meson')
```

Comment:
We should hardcode the path here otherwise a local installation for an app that hast $home access could potentially be used

Reply:
Should it be `/usr/bin/meson`? (from a quick which in build terminal)

Reply:
yes